profiles/serializers_test.py

```diff
@@ -304,3 +309,158 @@ def test_delete_employment(self):
 
         # Other profile is unaffected
         assert employment3.profile.work_history.count() == 1
+
+
+class ProfileFilledOutTests(ESTestCase):
```

Comment:
this question surely applies to other test classes in the codebase, but why is this inheriting from ESTestCase? is it testing any ES functionality? does it require indexing to be done?